From 240634b1afb968a051f8c68696eae2a582a02450 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Mon, 31 Aug 2020 20:16:25 +0200
Subject: [PATCH 1/2] Rename openpgm-5.2.pc.in

This will fix the following build failure:

config.status: error: cannot find input file: `openpgm-5.3.pc.in'

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 openpgm/pgm/{openpgm-5.2.pc.in => openpgm-5.3.pc.in} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename openpgm/pgm/{openpgm-5.2.pc.in => openpgm-5.3.pc.in} (100%)

--- a/openpgm/pgm/openpgm-5.2.pc.in
+++ /dev/null
@@ -1,12 +0,0 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
-
-Name: OpenPGM
-Description: PGM Protocol Library.
-Version: @PACKAGE_VERSION@
-# packagers may wish to move @LIBS@ to Libs.private for platforms with
-# versions of pkg-config that support static linking.
-Libs: -L${libdir} -lpgm @LIBS@
-Cflags: -I${includedir}/pgm-@VERSION_MAJOR@.@VERSION_MINOR@
--- /dev/null
+++ b/openpgm/pgm/openpgm-5.3.pc.in
@@ -0,0 +1,12 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: OpenPGM
+Description: PGM Protocol Library.
+Version: @PACKAGE_VERSION@
+# packagers may wish to move @LIBS@ to Libs.private for platforms with
+# versions of pkg-config that support static linking.
+Libs: -L${libdir} -lpgm @LIBS@
+Cflags: -I${includedir}/pgm-@VERSION_MAJOR@.@VERSION_MINOR@
